packages/nc-gui/components/workspace/CollaboratorsList.vue

```diff
@@ -56,67 +56,59 @@ onMounted(async () => {
     <div v-if="!filterCollaborators?.length" class="w-full h-full flex flex-col items-center justify-center mt-36">
       <Empty description="No members found" />
     </div>
-    <table v-else class="nc-collaborators-list-table !nc-scrollbar-md">
-      <thead>
-        <tr>
-          <th class="w-1/5">Users</th>
-          <th class="w-1/5">Date Joined</th>
-          <th class="w-1/5">Access</th>
-          <th class="w-1/5"></th>
-          <th class="w-1/5"></th>
-          <th></th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr v-for="(collab, i) of filterCollaborators" :key="i" class="relative w-full nc-collaborators">
-          <td class="!py-0 w-1/5 email">
+    <div v-else class="nc-collaborators-list !mt-10 nc-scrollbar-md rounded-md">
+      <div class="nc-collaborators-list-header bg-gray-50 rounded-t-md">
+        <div class="flex w-2/5 text-gray-600">Users</div>
+        <div class="flex w-2/5 text-gray-600">Date Joined</div>
+        <div class="flex w-2/5 text-gray-600">Access</div>
+        <!-- <div class="flex w-2/5 text-gray-600">Added By</div> -->
```

Comment:
why comment ?

Reply:
Rn we don't have provision to know who invited a person to ws/project. I had it discussed with @dstala

Reply:
https://github.com/nocodb/nocohub/issues/1271#issue-1914970453

Reply:
add the reason in the code as a comment as well. otherwise, people may think this is not in use and remove it.